### PR TITLE
Ensure that Stores locale is set

### PR DESCRIPTION
**Pull request by damianlegawiec**

### CODE REVIEW

File: backend/app/views/spree/admin/stores/_form.html.erb [hunk 1]
```diff
@@ -152,6 +152,7 @@
     <%= f.field_container :default_locale, class: ['form-group'] do %>
       <%= f.label :default_locale, Spree.t('i18n.language') %>
       <%= f.select :default_locale, options_from_collection_for_select(all_locales_options, :last, :first, @store.default_locale), {}, { class: 'select2' } %>
+      <%= f.select :default_locale, options_from_collection_for_select(all_locales_options, :last, :first, @store.default_locale || I18n.locale), {}, { class: 'select2' } %>
```
Comment (przemosk) [inline backend/app/views/spree/admin/stores/_form.html.erb:155]: @damianlegawiec should we add twice `default_locale` select ?

File: core/db/migrate/20210120142527_ensure_default_locale_in_spree_stores.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class EnsureDefaultLocaleInSpreeStores < ActiveRecord::Migration[6.0]
+  def change
+    Spree::Store.where(default_locale: nil).update_all(default_locale: I18n.locale)
```
Comment (przemosk) [inline core/db/migrate/20210120142527_ensure_default_locale_in_spree_stores.rb:3]: @damianlegawiec it's valid to check before update if that table exists in DB ?

File: backend/app/views/spree/admin/stores/_form.html.erb (hunk 1 above)
Comment (damianlegawiec) [reply backend/app/views/spree/admin/stores/_form.html.erb:155]: good catch - sth went wrong here ;)

File: core/db/migrate/20210120142527_ensure_default_locale_in_spree_stores.rb (hunk 2 above)
Comment (damianlegawiec) [reply core/db/migrate/20210120142527_ensure_default_locale_in_spree_stores.rb:3]: @przemosk table and column are created in previous migrations so no need IMHO

**Review by przemosk — APPROVED** [review]

**Pull request merged**